wai-aria/role/region-roles.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<html>
+<head>
+  <title>Region Role Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<!-- no label -->
+<nav role="region" data-testname="region without label" data-expectedrole="navigation" class="ex">x</nav>
```

Comment:
This is an interesting case. In Gecko, computedRole currently returns "region" here. While it's clear from the HTML AAM spec that `<section>` without a label shouldn't return "region", I'm not sure where it's specified that role="region" without a label shouldn't map to "region". Core AAM says that region without a label shouldn't be exposed as a landmark to individual platforms, but it says nothing about the fact that it shouldn't be exposed as a "region" as far as ARIA (not the platform APIs) are concerned. You can intuit the latter from the former, but unless I'm missing something, there's no spec concept which allows a role to be completely ignored/invalidated based on a missing name. There's "Name required true", but that's specified for button as well, and I'm not sure we'd want to completely ignore the role for an unlabelled button.

Reply:
Possibly a mistake. Since this PR is already merged, I filed your comment as a new issue:
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/28

Reply:
Closing the loop here too. The test is correct as written. 

> Certain landmark roles require names from authors. In situations where an author has not specified names for these landmarks, it is considered an authoring error. The user agent MUST treat such elements as if no [role](https://w3c.github.io/aria/#dfn-role) had been provided. If a valid fallback role had been specified, or if the element had an implicit ARIA role, then user agents would continue to expose that role, instead. Instances of such roles are as follows:
> 
> - [form](https://w3c.github.io/aria/#form)
> - [region](https://w3c.github.io/aria/#region)

Citation: https://w3c.github.io/aria/#document-handling_author-errors_roles